components/places/src/ffi.rs

```diff
@@ -255,6 +301,22 @@ impl PlacesConnection {
             Ok(())
         })
     }
+
+    fn wipe_local(&self) -> Result<()> {
+        self.with_conn(|conn| history::wipe_local(conn))
+    }
+
+    fn run_maintenance(&self) -> Result<()> {
+        self.with_conn(|conn| storage::run_maintenance(conn))
+    }
+
+    fn prune_destructively(&self) -> Result<()> {
+        self.with_conn(|conn| history::prune_destructively(conn))
+    }
+
+    fn delete_everything(&self) -> Result<()> {
```

Comment:
eek - I guess we should squeeze the word "history" in here too - but looking at the impl, the difference between `wipe_local` and `delete_everything` is tiny - I think this one will also prevent the history from coming back via sync - so for now, either here or in the IDL, move these close to each other and add an XXX comment that we should document exactly how these work.